roles/ansible-os-hardening/tasks/main.yml

```diff
@@ -20,6 +20,21 @@
     - '/bin'
     - '{{os_env_extra_user_paths}}'
 
+- name: activate gpg-check for yum-repos in yum.conf
+  lineinfile: dest='/etc/yum.conf' regexp='^gpgcheck=0' line='gpgcheck=1' backrefs=yes
```

Comment:
this may not work with indent. I think we may need to add a whitespace ^\s*gpgcheck=0